tox.ini

```diff
@@ -27,63 +26,22 @@ passenv =
     CHARM_BUILD_DIR
     MODEL_SETTINGS
 
-[testenv:fmt]
-description = Apply coding style standards to code
-deps =
-    black
-    isort
-commands =
-    isort {[vars]all_path}
-    black {[vars]all_path}
-
 [testenv:lint]
 description = Check code against coding style standards
-deps =
-    black
-    flake8==4.0.1
-    flake8-docstrings
-    flake8-copyright
-    flake8-builtins
-    pyproject-flake8
-    pep8-naming
-    isort
 commands =
     pflake8 {[vars]all_path}
     isort --check-only --diff {[vars]all_path}
     black --check --diff {[vars]all_path}
 
 [testenv:static]
 description = Run static analysis checks
-deps =
-    -r{toxinidir}/requirements.txt
-    mypy
-    types-PyYAML
-    pytest
-    pytest-operator
-    juju
-    types-setuptools
-    types-toml
 commands =
     mypy {[vars]all_path} {posargs}
 setenv =
     PYTHONPATH = ""
 
 [testenv:unit]
 description = Run unit tests
-deps =
-    pytest
-    coverage[toml]
-    -r{toxinidir}/requirements.txt
 commands =
-    coverage run --source={[vars]src_path} -m pytest --ignore {[vars]integration_test_path} -v --tb native -s {posargs}
+    coverage run --source={[vars]src_path} -m pytest -v --tb native -s {posargs}
     coverage report
-
-[testenv:integration]
-description = Run integration tests
-deps =
-    git+https://github.com/juju/python-libjuju.git
-    pytest
-    git+https://github.com/charmed-kubernetes/pytest-operator.git
-    -r{toxinidir}/requirements.txt
-commands =
-    pytest --asyncio-mode=auto -v --tb native --ignore {[vars]unit_test_path} --log-cli-level=INFO -s {posargs}
```

Comment:
Why are we removing integration test here?

Reply:
We didn't have any itests in this project anyway, so there's no point in keeping this. Just cleaning up.

Reply:
Got it, thanks :)